src/FakeItEasy/Creation/FakeAndDummyManager.cs

```diff
@@ -44,11 +44,14 @@ public FakeAndDummyManager(IDummyValueResolver dummyValueResolver, IFakeObjectCr
 
         public bool TryCreateDummy(Type typeOfDummy, LoopDetectingResolutionContext resolutionContext, out object? result)
         {
-            var creationResult = this.dummyValueResolver.TryResolveDummyValue(typeOfDummy, resolutionContext);
-            if (creationResult.WasSuccessful)
+            if (typeOfDummy != typeof(void))

```

Comment:
This isn't something that most users will ever notice, but wherever we add this rule, I think it warrants updating the list in https://fakeiteasy.github.io/docs/8.0.0/dummies/#how-are-the-dummies-made. As sad as making that list longer makes me. 😉